### ISSUE TITLE
[Question] Search Function

### ISSUE THREAD

**Issue by TurKurT656** [miteshpithadiya/SearchableSpinner#13]
Assume I have 2 texts inside the spinner: `"Good Morning"` and `"Hello World"`.
When I search with starting 'g' it works right. but when i search with starting 'w' it doesnt show anything. Am I wrong or all of you have this problem?

**Comment by teerarod** [comment]
Yes. I have similar problem. It return only when search with starting string.

**Comment by facundomedica** [comment]
Another problem that might be related to this is that when I have an array like this: ["testing something", "another test", "something else"]. If I search for "anothe" it finds "another test", but if I search "another" (notice the "R") it displays nothing.

**Comment by qingchen1016** [comment]
I have such a problem too, there are ways to solve the problem?
